README.md

```diff
@@ -6,6 +6,60 @@ Query API for the Tezos blockchain
 
 ## Running Conseil
 
+
+### Docker
+
+To build this project git clone it and run:
+
+```
+docker build -t conseil .
+```
+
+The Container can be configured using the following environment variables:
+
+Database Config:
+
+
+* DB_Host - default: db
+* DB_User - default: user
+* DB_Password - default: password
```

Comment:
Presuming the default credentials are going to be replaced by the credentials in the conseil.conf file?

Reply:
See the entrypoint script below. It replaces these with env vars.